src/libs/language-javascript-latest/_babelHelpers.js

```diff
@@ -0,0 +1,84 @@
+
+//
+// Transpiling with babel is needed because describeApi will directly
+// call `require(PROJECT_ROOT + '/api');
+//
+
+import path from 'path';
+import { stripIndent } from 'common-tags';
+import { debug, warning } from '../../logger';
+
+// Support for `babel` property in project package.json to extend base babel config
+// to add plugins and more presets, etc. React, JSX Transform, Syntax support
+const babelRequiredPkgJson = require(path.join(process.cwd(), 'package.json'));
+
+// check if babel key is package.json
+const hasBabelConfigInPkgJson = ('babel' in babelRequiredPkgJson);
+
+if (hasBabelConfigInPkgJson) {
```

Comment:
a warning now asks users to make sure they're including the correct plugins and presets